documentation/book/cluster-operator.adoc

```diff
@@ -107,7 +127,15 @@ A JSON string representing the storage configuration for the Zookeeper nodes. Se
 `zookeeper-metrics-config`::
 A JSON string representing the JMX exporter configuration for exposing metrics from Zookeeper nodes.
 When this field is absent no metrics will be exposed.
- eans having no metrics exposed.
+`zookeeper-logging`::
+A JSON string representing the level of logging for Zookeeper nodes. This field can contain JSON with loggers, which should be changed and levels.
+Or there can be specified name of config map, which should be used for setting logger levels.
+When this field is absent default values are used.
```

Comment:
I noticed that we list what the default values for other descriptions in the documentation. Should we list the default values here as well? Or would the default JSON string be too large?

Reply:
Yeah, I was thinking about that too. I did not want to spam docs by copy paste all the levels at several places.